care/facility/models/prescription.py

```diff
@@ -148,6 +165,7 @@ class MedicineAdministration(BaseModel):
         on_delete=models.PROTECT,
         related_name="administrations",
     )
+    dosage = models.CharField(max_length=100, blank=True, null=True)
```

Comment:
Should we consider populating this with the `base_dosage` value from the prescription for non-titrated prescriptions?

```suggestion
    dosage = models.CharField(max_length=100)
```